backend/entities/task.py

```diff
@@ -17,4 +17,11 @@ def create(body):
         "detail": detail
         }
     table.put_item(Item=item)
-    return json.dumps(item)
\ No newline at end of file
+    return json.dumps(item)
+
+def get():
+    response = table.scan()
+    items = response['Items']
+    return json.dumps(items)
```

Comment:
https://admin-it.xyz/python/flask-jsonify-jsondumps-difference/
https://qiita.com/nagataaaas/items/24e68a9c736aec31948e